gcc/testsuite/jit.dg/test-used-attribute.c

```diff
@@ -0,0 +1,114 @@
+/* { dg-do compile { target x86_64-*-* } } */
+
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "libgccjit.h"
+
+/* We don't want set_options() in harness.h to set -O3 to see that the pure
+   attribute affects the optimizations. */
```

Comment:
Maybe something else than "optimizations" here?

Reply:
Let's remove the comment...